print_resc_properties.py

```diff
@@ -0,0 +1,12 @@
+from irods.resource import iRODSResource
+from irods.models import Resource
+import irods.test.helpers as h
+import sys
+
+s = h.make_session()
+
+r = s.resources.get(sys.argv[1])
+
+print('id = ',r.parent_id)
+print('name = ',r.parent_name)
+print('resc_hier = ',r.hierarchy)
```

Comment:
when you write the test - please confirm the string-ness and int-ness of the return types.

Reply:
will do!

Reply:
@trel good call. it did seem under some circumstances the id returned in the result set was a string